SpecializedServers/Horizon/SERVER/Medius/MAPS.cs

```diff
@@ -283,6 +306,35 @@ public static string ShiftString(string t)
             return t[1..] + t[..1];
         }
 
+        public static ulong ReverseBytesULong(ulong value)
```

Comment:
Theses methods are present in the endianutils library, no need for duplicates.

Reply:
Yes, this was before you told me about EndianUtils library.